CHANGELOG.md

```diff
@@ -1,4 +1,23 @@
 # Changelog
+### 2.22.0
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.8.12](https://fluentbit.io/announcements/v1.8.12/)
+* Amazon CloudWatch Logs for Fluent Bit 1.7.0
+* Amazon Kinesis Streams for Fluent Bit 1.8.1
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.21.6` this release adds:
+* Feature - Add support for external_id in AWS output plugins [aws-for-fluent-bit:171](https://github.com/aws/aws-for-fluent-bit/issues/171)
+* Bug - Fix truncation issue after compression [kinesis:183](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit/issues/183)
+
+Same as `2.21.6`, this release includes the following fixes for AWS customers that has been accepted by upstream:
```

Comment:
grammarL "that **have** been accepted"

Reply:
Thanks

Reply:
Updates: "this release includes the following **fix** for AWS customers that **has** been accepted by upstream:"